lib/commands.js

```diff
@@ -264,10 +264,16 @@ module.exports = (message, command, args) => {
                 m.addRole(role).catch(e => log.debug(e))
 
                 // Add the 'contributed' role to user if they have contributed
-                if (contributors[body.id]) {
+                if (contributors[body.id] && contributors[body.id][config.github.contributedRole]) {
```

Comment:
This can be simplified to loop over contributors[body.id] and using key as role search